rust/private/rustc.bzl

```
@@ -299,7 +299,11 @@ def _get_dir_names(files):
 def add_crate_link_flags(args, dep_info):
     # nb. Crates are linked via --extern regardless of their crate_type
     args.add_all(dep_info.direct_crates, map_each = _crate_to_link_flag)
-    args.add_all(dep_info.indirect_crates, map_each = _get_crate_dirname, uniquify = True, format_each = "-Ldependency=%s")
+    args.add_all(
+        depset(transitive=[dep_info.indirect_crates, dep_info.direct_crates]),
```

This can be dep_info.transitive_crates, and then indirect_crates can be removed

---

```suggestion
        dep_info.transitive_crates,
```
(fancy github features :))

---

:) I actually suggested a month ago to build a chrome extension to do a "Suggested Fix" feature and then Github released it :) (we have a similar feature inside Google)